File: src/WebApp/src/app/invoice/interfaces/response-invoice-sale.ts
import {InvoiceSaleDetail} from './invoice-sale-detail';
import {TributoSale} from './tributoSale';
import {InvoiceSaleAccount} from './invoice-sale-account';
import {InvoiceSale} from './invoiceSale';

// respuesta del servidor
// comprobante de venta detallado.
export interface ResponseInvoiceSale {
  invoiceSale: InvoiceSale;
  invoiceSaleDetails: Array<InvoiceSaleDetail>;
  tributoSales: Array<TributoSale>;
  invoiceSaleAccounts: Array<InvoiceSaleAccount>;
}

File: src/Nebula/Data/ViewModels/ResponseInvoiceSale.cs
using System.Collections.Generic;
using Nebula.Data.Models;

namespace Nebula.Data.ViewModels
{
    public class ResponseInvoiceSale
    {
        public InvoiceSale InvoiceSale { get; set; }
        public List<InvoiceSaleDetail> InvoiceSaleDetails { get; set; }
        public List<TributoSale> TributoSales { get; set; }
        public List<InvoiceSaleAccount> InvoiceSaleAccounts { get; set; }
    }
}
